app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def show
+    @user = User.find_by(id: params[:id])
+    return if @user
+
+    redirect_to root_path, alert: t("sessions.show.user_not_found")
+  end
```

Comment:
```suggestion
  def show; end
```
E đưa logic bên trong ra hàm load_user và gọi trong before_action nha